commit 7145bd4d6edd1622585d7b18b8bba9a71cd41e2b
Author: Mark Wielaard <mark@klomp.org>
Date:   Tue May 22 22:12:11 2018 +0200

    find-debuginfo.sh: Handle application/x-pie-executable (file 5.33 output).
    
    A new version of file (5.33) changed the output for PIE executables.
    Which are now represented as application/x-pie-executable; charset=binary.
    
    The following change simply recognizes that binary format as one for
    which -g applies. This fixes accidental stripping of the .symtab in
    glibc (which use -g to keep that symbol table).
    
    https://bugzilla.redhat.com/show_bug.cgi?id=1581224
    
    Signed-off-by: Mark Wielaard <mark@klomp.org>

diff --git a/scripts/find-debuginfo.sh b/scripts/find-debuginfo.sh
index a7c2db0..90a4494 100755
--- a/scripts/find-debuginfo.sh
+++ b/scripts/find-debuginfo.sh
@@ -235,6 +235,7 @@ strip_to_debug()
   $strip_g && case "$(file -bi "$2")" in
   application/x-sharedlib*) g=-g ;;
   application/x-executable*) g=-g ;;
+  application/x-pie-executable*) g=-g ;;
   esac
   eu-strip --remove-comment $r $g ${keep_remove_args} -f "$1" "$2" || exit
   chmod 444 "$1" || exit
